src/distributions/microsoft/models.ts

```diff
@@ -0,0 +1,15 @@
+export type OsVersions = 'linux' | 'macos' | 'windows';
+export type ArchiveType = 'tar.gz' | 'zip';
```

Comment:
Could you please remove the export keyword because these types are only used inside `models.ts`